docs/package_integration/guide.md

```diff
@@ -0,0 +1,71 @@
+# UDS Marketplace Package Integration Guide
+
+## Introduction
+
+This guide is intended for developers integrating applications with UDS (Unicorn Delivery Service). It provides an overview of the integration process, key considerations, and resources to ensure a smooth integration.
+
+Integrating a Package fundamentally means:
+1. Creating a repository `uds-package-<name>`
+2. Integrating the upstream helm chart as a zarf package `zarf.yaml` to build a declarative OCI artifact
+3. Adding a UDS package Custom Resource `uds-package.yaml` to integrate with UDS Core via Pepr
+4. Build a 'zero CVE' package by replacing images with a `*-unicorn` flavored image
+
+## Prerequisites
+
+- UDS generally assumes applications are containerized and can be deployed via Kubernetes. (exceptions are possible, but rare, and require additional work e.g. KubeVirt)
+- The integrator (you) should have access to the application image and deployment code.
+- The "App" should have a compatible license for UDS integration (Apache 2.0 or similar) [see Licensing Considerations](#licensing-considerations).
+
+### Licensing Considerations
+
+- Defense Unicorns uses Apache 2.0 licensed products exclusively, see the [Open Source Policy](https://github.com/defenseunicorns/uds-common/blob/main/docs/adrs/0002-apache-2.0-for-all-uds-products.md).
```

Comment:
```suggestion
- Defense Unicorns creates Apache 2.0 licensed products exclusively, see the [Open Source Policy](https://github.com/defenseunicorns/uds-common/blob/main/docs/adrs/0002-apache-2.0-for-all-uds-products.md).
```
We use AGPL/MPL products inside some packages - just our own stuff is only Apache 2